Assets/Scripts/GameElements/PlayerController.cs

```diff
@@ -413,4 +453,31 @@ private void SetInvulnerable()
         invulnerabilityTime = invulnerabilityInitTime;
         invulnerable = true;
     }
+
+    public bool IsTrapped()
+    {
+        return isPlayerTrapped;
+    }
+
+    public void TrapPlayer()
+    {
+        isPlayerTrapped = true;
+        
+        gripDrainScalar = gripDrainScalar * gripAmplifyingFactor;
+        innerWingForce = innerWingForce * wingDampeningFactor;
+        outerWingForce = outerWingForce * wingDampeningFactor;
+        lateralWingForce = lateralWingForce * wingDampeningFactor;
```

Comment:
I suppose this is fine, but I would prefer if `gripDrainScalar` and all the other forces were their own private variables rather than reassigning them. Then you could make the variables that we define in the editor something like `initGripDrainScalar`

Reply:
ahhh gotcha